### ISSUE TITLE
.btn btn-danger has a blue link and is not accessible

### ISSUE THREAD

**Issue by mrmjprice** [wmcadigital/wmn-cms#21]
<img width="121" alt="Screenshot 2021-02-19 at 10 49 51" src="https://user-images.githubusercontent.com/10854265/108495044-32f23500-72a0-11eb-823e-e6aa59f20e75.png">

Example: Delete all button of SEO Checker > Issues > Validation issues

**Comment by catiarodriguescosta** [comment]
Fixed on localhost! 
Will be live in the next release! 👍
